google/resource_compute_instance.go

```diff
@@ -741,6 +753,54 @@ func expandComputeInstance(project string, d *schema.ResourceData, config *Confi
 	}, nil
 }
 
+func getAllStatusExcept(exceptedStatus string) []string {
```

Comment:
In fact, this is not an expected status passed as parameter but rather an excepted status :smile:. In the context of this method, there is no notion of "expected", but just a status that we want to except. I should have used the signature `getAllStatusBut(status string)` to avoid any confusion between `expected` and `excepted`, my mistake :smile:. I'm going to rename it and add a comment :+1: